test/unit/test_assembly.py

```diff
@@ -312,6 +312,18 @@ def test_multi_overlap(self):
             str(Bio.SeqIO.read(outFasta, 'fasta').seq),
             str(Bio.SeqIO.read(expected, 'fasta').seq))
 
+class TestGap2Seq(TestCaseWithTmp):
+    '''Test gap-filling tool Gap2Seq'''
+
+    def test_gapfill(self):
+        from os.path import join
+        inDir = util.file.get_test_input_path()
+        in_scaffold = join(inDir, 'TestOrderAndOrient', 'expected.ebov.doublehit.fasta')
+        with util.file.tempfname(suffix='.filled.fasta') as filled:
+            assembly.gapfill_gap2seq(in_scaffold=in_scaffold,
+                                     inBam=join(inDir, 'G5012.3.testreads.bam'), out_scaffold=filled)
+            self.assertEqualContents(filled, join(inDir, 'TestGap2Seq', 'expected.ebov.doublehit.gapfill.fasta'))
```

Comment:
I remember, but it's a small file, and there's some benefit to being able to look at assertions and see _why_ they should hold.  E.g. here one can align reads to the filled-in scaffold and see that they do align.